README.md

```diff
@@ -24,18 +24,22 @@ Run app.py to launch Timelord
 
 #### User Commands:
 - `/help` Get this list of commands
-- `/timelog` Opens a time logging form
+- `/timelog` Open a time logging form
 - `/deletelast` Delete your last entry
-- `/myentries n` Get a table with your last n entries (defaults to 5)
+- `/myentries n` Get your last n entries (defaults to 5)"""
```

Comment:
What is the """ doing here?

Reply:
Mistake, probably left over from copy pasting the /help command and being lazy.